css-brochure-form/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+}
+
+h2 {
+  text-align: center;
+  margin-top: 90px;
+  color: #ec5d57;
+  font-size: 22px;
+}
+
+input {
+  border-radius: 6px;
+  border: 2px solid lightgrey;
+  margin-left: 700px;
+  width: 500px;
+  padding: 7px;
+  font-size: 16px;
+  margin-bottom: 15px;
+}
+
+label {
+  margin-left: 718px;
+  font-size: 14px;
+}
+
+select {
+  border-radius: 6px;
+  border: 2px solid lightgrey;
+  justify-content: center;
+  margin-left: 700px;
+  width: 500px;
+  padding: 10px;
+  font-size: 16px;
+  margin-bottom: 15px;
+  text-align: start;
+  background-color: #f3f3f3;
+}
+
+div > input:nth-child(1) {
```

Comment:
Do we have more than one `input` element inside of `div` which has `input` as child?